korlib/CMakeLists.txt

```diff
@@ -38,6 +38,7 @@ include_directories(${PYTHON_INCLUDE_DIRS})
 include_directories(${Vorbis_INCLUDE_DIR})
 
 add_library(_korlib SHARED ${korlib_HEADERS} ${korlib_SOURCES})
+set_target_properties(_korlib PROPERTIES PREFIX "")
```

Comment:
By default, the resulting library (on Linux anyways) was named `lib_korlib.so`, which cannot be imported with `from _korlib import *`. This makes it output a `_korlib.so` file.